src/common/HTMLFormatter.h

```diff
@@ -24,6 +24,8 @@ namespace ceph {
 
     /* with attrs */
     void dump_string_with_attrs(std::string_view name, std::string_view s, const FormatterAttrs& attrs) override;
+  protected:
+    bool m_pretty;
```

Comment:
copying @cbodley's comment from the original PR:
`HTMLFormatter` inherits from `XMLFormatter`, which already has a `const bool m_pretty`

Reply:
ack, on current version testing, the _pretty_ modification is not necessary